cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -480,6 +480,50 @@ a|
 Terminate transaction now allows `YIELD`.
 The `WHERE` clause is not allowed on its own, as it is for `SHOW`, but needs the `YIELD` clause.
 
+a|
+label:functionality[]
+label:updated[]
+[source, cypher, role="noheader"]
+----
+SHOW TRANSACTIONS [transaction-id[,...]]
+----
+[source, cypher, role="noheader"]
+----
+TERMINATE TRANSACTIONS transaction-id[,...]
+----
+a|
+The `transaction-id` now allows general expressions resolving to a string or a list of strings instead of just parameter.
```

Comment:
The `transaction-id` now allows general expressions resolving to a string or a list of strings instead of just parameters.